src/main/java/uk/gov/hmcts/reform/pip/rules/errorhandling/GlobalExceptionHandler.java

```diff
@@ -32,4 +35,21 @@
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
 
+    /**
+     * Template exception handler, that handles a custom ParseException,
+     * and returns a 500 in the standard format.
+     * @param ex The exception that has been thrown.
+     * @return The error response, modelled using the ExceptionResponse object.
+     */
+    @ExceptionHandler(ParseException.class)
+    public ResponseEntity<ExceptionResponse> handleParseException(
+        ParseException ex) {
+
+        ExceptionResponse exceptionResponse = new ExceptionResponse();
+        exceptionResponse.setMessage("Failed to parse date");
```

Comment:
we shouldnt hard code the value here, it should be a variable that is accessed here